kafka-rest/src/test/java/io/confluent/kafkarest/integration/ClusterTestHarness.java

```diff
@@ -89,6 +89,8 @@
 public abstract class ClusterTestHarness {
 
   private static final Logger log = LoggerFactory.getLogger(ClusterTestHarness.class);
+  private static final long ONE_SECOND_MS = 1000l;
```

Comment:
super-nit: I think `L` is preferable over `l` as the latter kinda looks like a `1`.